scripts/openeuler/packer/README.md

```diff
@@ -0,0 +1,8 @@
+# Install cloud-init and other tools
+
+Scripts in this folder should be running in the 'VM/Instance' launched by packer.
+
+```
+ami-install-cloud-init.sh - Install cloud-init and other tools for built AMI image.
+qemu-install-cloud-init   - Install cloud-init and other tools for built qcow2 image.
```

Comment:
The script name should be `qemu-install-cloud-init.sh`

Reply:
Updated.